src/en/getting-started.md

```diff
@@ -4,80 +4,90 @@ TODO: Mediawiki status needs new screenshot when status has been updated in Juju
-[general install instructions here](./getting-started-general.html).
+See the [general Getting Started page][getting-started-general] if you're using
+something other than Xenial.
 
-To get the best experience, as well as Juju, this guide will also set up:
+Apart from Juju, the following technologies will be used:
    
-- LXD - a hypervisor for LXC, providing fast, secure containers.
-- ZFS - a combined filesystem/LVM which gives great performance.
+- [LXD][LXD-upstream]: a hypervisor for LXC, providing fast, secure containers.
+- [ZFS][ZFS-wiki]: a highly performant and feature-rich filesystem and logical volume manager.
 
-Both the above are provided with Ubuntu 16.04LTS.
+[Xenial-download]: http://www.ubuntu.com/download/ "Xenial download"
+[getting-started-general]: ./getting-started-general.html "general Getting Started"
+[LXD-upstream]: https://linuxcontainers.org/lxd/ "LXD upstream"
+[ZFS-wiki]: https://wiki.ubuntu.com/ZFS "ZFS Ubuntu wiki"
 
 
 ## Install the software
 
-Run the following commands to install the required software:
+Begin by installing the required software:
 
 ```no-highlight
 sudo apt update
 sudo apt install juju zfsutils-linux
 ```
 
-## Initialise LXD
+## Groups and LXD initialisation 
 
-In order to use LXD, your user must be in the 'lxd' group. All system users are
-automatically added to this group, but you may need to refresh the current 
-session. You can confirm your user is part of this group by running the command:
+Firstly, in order to use LXD, your user must be a member of the `lxd` group.
+This should already be the case but you can confirm this by running the
+command:
 
 ```bash
 groups
 ```
 
-This should indicate the user is a member of the lxd group, amongst others (your
-groups may vary from these):
+Sample output is provided below:
 
 ```no-highlight
 lxd adm cdrom sudo dip plugdev lpadmin sambashare ubuntu
 ```
 
-If the `lxd` group is not present, you can refresh group membership with the 
-command:
-  
+Your groups may vary, but if `lxd` is absent you should refresh group
+membership with:
+
 ```bash
 newgrp lxd
 ```
 
-LXD includes an interactive initialisation which will also set up a ZFS pool 
-to use and configures networking for your containers. To start this process, 
-enter:
+Secondly, LXD includes an interactive initialisation which includes setting up
+a ZFS pool and appropriate networking for your LXD containers. To start this
+process, enter:
 
 ```bash
 sudo lxd init
 ```
 
-You will be prompted for various options. As an example, to configure LXD to 
-create a new 32GB ZFS pool to use, called 'lxd-pool', and set up a bridge 
-network (required for Juju), your session would look like this:
+You will be asked several questions. In the example below, LXD will i) create a
+32GB ZFS pool, ii) refrain from putting the pool on a separate block device,
+iii) refrain from listening over the network, and iv) trigger the setup of a
+bridge network (required for Juju).
+
```

Comment:
I haven't seen us use the lower-case Roman numeral style before. I don't think it is bad, but since it appears to be unique I thought it should be highlighted and considered before merging.